.github/workflows/pr-close.yml

```diff
@@ -2,6 +2,8 @@ name: Pull Request Closed
 
 on:
   pull_request:
+    branches:
```

Comment:
We don't want to limit this to just the main branch, because that'll skip cleanup.